network/p2p/dns/resolver_test.go

```diff
@@ -76,33 +75,35 @@ func TestResolver_CacheExpiry(t *testing.T) {
 		metrics.NewNoopCollector(),
 		dnsCache,
 		WithBasicResolver(&basicResolver),
-		WithTTL(1*time.Second)) // cache timeout set to 1 seconds for this test
+		WithTTL(3*time.Second)) // cache timeout set to 3 seconds for this test
 
 	cancelCtx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 	ctx, _ := irrecoverable.WithSignaler(cancelCtx)
 	resolver.Start(ctx)
 	unittest.RequireCloseBefore(t, resolver.Ready(), 100*time.Millisecond, "could not start dns resolver on time")
 
-	size := 10 // we have 10 txt and 10 ip lookup test cases
-	times := 5 // each domain is queried for resolution 5 times
+	size := 5  // we have 5 txt and 5 ip lookup test cases
```

Comment:
why were these values reduced?

Reply:
To reduce the test complexity for debugging and also reduce the run time. The goal is to test the concurrent queries, and 30 test cases ( (5+5) * 3) serve the same purpose as the original 100 test cases ((10 + 10) * 5), but with a smaller set of goroutines. Ultimately the desire is to even reduce it further to 5-10 concurrent goroutines for the test.